packages/agent-toolkit/src/core/tools/platform-api-tools/workforms-tools/workforms.consts.ts

```diff
@@ -28,18 +31,43 @@ export const GraphQLDescriptions = {
       features: 'Object containing feature toggles and settings like password protection, response limits, etc.',
       appearance: 'Object containing visual styling settings including colors, fonts, layout, and branding.',
       accessibility: 'Object containing accessibility settings such as language, alt text, and reading direction.',
-      tags: 'Array of tracking tags for categorization and analytics (e.g., UTM parameters for marketing tracking).',
+      tags: {
+        description:
+          'Array of tracking tags for categorization and analytics (e.g., UTM parameters for marketing tracking).',
+        id: 'The unique identifier for the tag. This will get auto generated when creating a tag and can’t be updated. This is required when updating or deleting a tag.',
+        name: 'The name of the tag. This can only be created, not updated. This is required when creating a tag.',
+        value: 'The value of the tag. This value is required when creating or updating a tag.',
+        columnId:
+          'The ID of the column this tag is associated with. This will get auto generated when creating a tag and can’t be updated.',
+      },
     },
     inputs: {
-      title: 'The title text for the form. Must be at least 1 character long.',
-      description: 'Optional description text providing context about the form purpose.',
+      title:
+        'The title text for the form. Must be at least 1 character long. Can only be updated if the update action is updateFormHeader.',
+      description:
+        'Optional description text providing context about the form purpose. Can only be updated if the update action is updateFormHeader.',
       input: 'Complete form configuration object containing properties to create or update.',
-      questions: 'Ordered array of question IDs for reordering. Must include all existing question IDs.',
+      questions:
+        'Ordered array of dehydrated questions, object only including each question ID, for reordering. Must include all existing question IDs. Required if the update action is updateQuestionOrder.',
+      questionId: 'The unique identifier for the question. Used to target specific questions within a form.',
```

Comment:
I saw this is used for delteion dor instance. since its optional param, this needs to also specify when this is mandatory. this is relevant for all descriptions :)